pandas/core/generic.py

```diff
@@ -4311,60 +4311,67 @@ def _needs_reindex_multi(self, axes, method, level):
     def _reindex_multi(self, axes, copy, fill_value):
         return NotImplemented
 
-    _shared_docs['reindex_axis'] = ("""Conform input object to new index
-        with optional filling logic, placing NA/NaN in locations having
-        no value in the previous index. A new object is produced unless
-        the new index is equivalent to the current one and copy=False.
+    _shared_docs['reindex_axis'] = ("""
+        Conform input object to new index.
+
+        By default, places NaN in locations having no value in the
+        previous index. A new object is produced unless the new index
+        is equivalent to the current one and copy=False.
 
```

Comment:
`reindex_axis` is a deprecated method, so it would be good to add a 

```
.. deprecated:: 0.21.0
    Use `reindex` instead.

Reply:
Thanks, I have added this to the end of the summary. Based on other docstrings it looks like that's the place to put it.

Reply:
Yes, that looks good!